_includes/projects.html

```diff
@@ -3,7 +3,7 @@ <h2>Projects</h2>
   <div class="row">
     {% for item in site.data.projects %}
     <div class="col">
-      <a
+      <a href="/projects/{{ item.page-name }}"
```

Comment:
Adding the href here doesn't make a difference except changing the colour of the text. I think the href could be removed here.

Reply:
Hi, @anandrajaram21. I did that based on a Google Lighthouse recommendation which said Google couldn't crawl the link without a href. 

Thought we'd want the project pages to be indexed for SEO so added it (and it means non-JS users will be able to see the project page (since the modal is JS only).

Happy to remove it though if you prefer.

Reply:
Technically, if you want it to be a 'link' that doesn't actually link to another page, you should use the `<button>` tag instead of the `<a>` tag. There is some discussion here: https://css-tricks.com/use-button-element/#lets-consider-if-a-button-doesnt-have-a-meaningful-href-its-a-button